RecoParticleFlow/PFProducer/plugins/PFProducer.cc

```diff
@@ -491,75 +485,29 @@ PFProducer::beginRun(const edm::Run & run,
 
 
 void 
-PFProducer::produce(Event& iEvent, 
-		    const EventSetup& iSetup) {
-  
-  LogDebug("PFProducer")<<"START event: "
-			<<iEvent.id().event()
-			<<" in run "<<iEvent.id().run()<<endl;
-  
-
-  // Get The vertices from the event
-  // and assign dynamic vertex parameters
-  edm::Handle<reco::VertexCollection> vertices;
-  bool gotVertices = iEvent.getByToken(vertices_,vertices);
-  if(!gotVertices) {
-    ostringstream err;
-    err<<"Cannot find vertices for this event.Continuing Without them ";
-    LogError("PFProducer")<<err.str()<<endl;
-  }
+PFProducer::produce(Event& iEvent, const EventSetup& iSetup)
+{
+  LogDebug("PFProducer")<<"START event: " <<iEvent.id().event() <<" in run "<<iEvent.id().run()<<endl;
 
   //Assign the PFAlgo Parameters
-  pfAlgo_->setPFVertexParameters(useVerticesForNeutral_,vertices.product());
```

Comment:
Same as for `PFEGammaProducer`; I think the logging before is not necessary because `product()` throws an exception already.